test/nlpmodels.jl

```diff
@@ -0,0 +1,207 @@
+using ADTypes: ADTypes
+using LinearAlgebra
+using SparseArrays
+using SparseConnectivityTracer
+import SparseConnectivityTracer as SCT
+using Test
+
+using Pkg
+Pkg.add([
+    "ADNLPModels", "ForwardDiff", "OptimizationProblems", "NLPModels", "NLPModelsJuMP"
+])
```

Comment:
Why is this in here?

Reply:
To avoid adding heavy test dependencies, so that the standard test suite remains quick.
Do you want me to run this extended suite with a label, like the benchmarks ?

Reply:
> To avoid adding heavy test dependencies, so that the standard test suite remains quick.

I think the new-ish caching action should make it acceptable to add the packages to the test env?

> Do you want me to run this extended suite with a label, like the benchmarks ?

Let's try it the way it is now. 👍

Reply:
I myself don't want to load jump in my test env every time. I'd rather merge it with the pkg shenanigans

Reply:
Unlike you I'm one of the brave who test locally 😉